app/assets/javascripts/initializers/initializeDrawerSliders.js

```diff
@@ -33,19 +32,15 @@ function initializeDrawerSliders() {
 }
 
 function listenForNarrowMenuClick(event) {
-  const navLinks = document.getElementsByClassName('narrow-nav-menu');
+  const navLinks = document.getElementById('narrow-nav-menu');
   const narrowFeedButt = document.getElementById('narrow-feed-butt');
-  for (let x = 0; x < navLinks.length; x++) {
-    document.getElementById('narrow-nav-menu').classList.remove('showing');
-  }
+    navLinks.classList.remove('showing');
   if (narrowFeedButt) {
-    narrowFeedButt.addEventListener('click', function() {
-      document.getElementById('narrow-nav-menu').classList.add('showing');
-    });
-  }
-  for (let i = 0; i < navLinks.length; i++) {
-    navLinks[i].addEventListener('click', function(event) {
-      document.getElementById('narrow-nav-menu').classList.remove('showing');
+    narrowFeedButt.addEventListener('click', () => {
+      navLinks.classList.add('showing');
     });
   }
+  navLinks.addEventListener('click', (event) => {
+    navLinks.classList.remove('showing');
```

Comment:
Inside the event listener, you will have access to the nav link through `event.target` property.

```suggestion
    event.target.classList.remove('showing');
```

Reply:
Fixed it. Thanks for the tips :)